youtube_dl/extractor/abc.py

```diff
@@ -103,6 +103,61 @@ def _real_extract(self, url):
         }
 
 
+class ABCIViewShowIE(InfoExtractor):
+    IE_NAME = 'abc.net.au:iview:show'
+    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*show/(?P<id>[^/?#]+)/?'
```

Comment:
```suggestion
    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*show/(?P<id>[^/?#]+)/?$'
```
Adding a '$' to the end of the _VALID_URL will fix your failing test - where it also matches an ABCIViewIE URL.

Reply:
Yeah - I got asked to take the `$` off it in review. Wasn't really sure why and haven't had a spare minute to look at it.

Reply:
Added it back in, seems to have fixed the tests.